### PR TITLE
greentext code improvement

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Small code improvement PR to bring greentext code up to 'modern' standards.

## Why It's Good For The Game

Not the game but just generally better code, since I didn't like how it was previously.

## Changelog

Not needed.

### CODE REVIEW

File: code/modules/events/wizard/greentext.dm [hunk 1]
```diff
@@ -84,19 +80,27 @@
 		return QDEL_HINT_LETMELIVE
```
Comment (Mothblocks) [inline code/modules/events/wizard/greentext.dm:80]: i beg you please kill this, i hate all the lavaland or wahtever crap that does this

Comment (Mothblocks) [reply code/modules/events/wizard/greentext.dm:80]: make the object indestructible or wahtever instead i dont care. just remove Destroy needing fucking `force` in order to actually do what it says

Comment (JohnFulpWillard) [reply code/modules/events/wizard/greentext.dm]: ok done did it yea its completed yes

**Comment by JohnFulpWillard** [comment]
NOOOOOOOOOOOOO

**Pull request merged**